src/planner/operator/logical_in_query_call.cpp

```diff
@@ -9,6 +9,7 @@ void LogicalInQueryCall::computeFlatSchema() {
     for (auto& outputExpression : outputExpressions) {
         schema->insertToGroupAndScope(outputExpression, 0);
     }
+    schema->insertToGroupAndScope(rowIDExpression, 0);
```

Comment:
open an issue for me to somehow remove rowIDExpression from LogicalInQueryCall, LogicalScanFile, ...